crates/bitwarden-uniffi/src/platform/fido2.rs

```diff
@@ -216,6 +246,8 @@ pub trait Fido2CredentialStore: Send + Sync {
         rip_id: String,
     ) -> Result<Vec<CipherView>, Fido2CallbackError>;
 
+    async fn all_credentials(&self) -> Result<Vec<CipherView>, Fido2CallbackError>;
```

Comment:
Instead of adding a separate function, could we update `find_credentials` to make `rp_id` optional just like the `ids` list is?

Then we could also have `silently_discover_credentials` take an optional `rp_id` which would save us from having a mostly duplicate `credentials_for_autofill` function and error type.

Reply:
Ah okay I didn't know that was part of the spec, I though it was just a utility function for our clients. That makes sense then!